workflow/lib/shared/initialize_paramsearch.py

```diff
@@ -0,0 +1,173 @@
+"""Initialize parameter searches."""
+
+from snakemake.utils import Paramspace
+import pandas as pd
+from itertools import product
+from pathlib import Path
+
+from lib.shared.file_utils import get_filename
+
+def initialize_segment_sbs_paramsearch(config):
+    """Initialize parameter search for sbs segmentation.
+
+    Args:
+        config (dict): Configuration dictionary.
+
+    Returns:
+        dict: Updated configuration dictionary.
+        list: Nuclei diameters.
+        list: Cell diameters.
+        list: Flow thresholds.
+        list: Cell probabilities.
+        dict: Output patterns.
+
+    """
+
```

Comment:
This throws an error during linting with `ruff check`. Are you able to check these issues and address? There are a couple others.

Reply:
Ignored issues in notebooks in https://github.com/cheeseman-lab/brieflow/pull/23/commits/d28014c69a095463e5af00e10b8b0c08d340c21c.

Also reformatted with ruff